fvm/src/machine/boxed.rs

```diff
@@ -60,13 +61,13 @@ impl<M: Machine> Machine for Box<M> {
     }
 
     #[inline(always)]
-    fn into_store(self) -> Self::Blockstore {
-        (*self).into_store()
+    fn flush(&mut self) -> Result<Cid> {
+        (**self).flush()
```

Comment:
? clippy or something cause this re-ordering?

Reply:
Implementation order now follows trait declaration order.